services/service.ts

```diff
@@ -0,0 +1,341 @@
+import fs from 'fs';
+import { createHash } from 'crypto';
+import fetch from 'node-fetch';
+import path from 'path';
+import child_process from 'child_process';
+
+// @ts-ignore
+import { Promise } from 'bluebird';
+import progress from 'progress-stream';
+import { Service, Message } from 'webos-service';
+
+import serviceInfo from './services.json';
+import { asyncAccess, asyncExecFile, asyncPipeline, asyncUnlink, asyncWriteFile } from './adapter';
+import { makeError, makeSuccess } from './protocol';
+import ServiceRemote from './webos-service-remote';
+
+const kHomebrewChannelPackageId = serviceInfo.services[0].name;
```

Comment:
This is going to be equal to `org.webosbrew.hbchannel.service`, while in fact should be `org.webosbrew.hbchannel`. I have no idea how to properly extract this yet.

Reply:
Ah good catch! Used `app-info.json`, hope that makes sense.